src/klipse/compiler.cljs

```diff
@@ -104,21 +104,21 @@
           :no-pr-str-on-value true
           :context (or context :statement)}))
 
-(defmethod core-eval :replumb [s {:keys [static-fns context verbose external-libs] :or {static-fns false context nil external-libs nil}} cb]
+(defmethod core-eval :replumb [s {:keys [preamble static-fns context verbose external-libs] :or {preamble nil static-fns false context nil external-libs nil}} cb]
   (let [opts (build-repl-opts {:static-fns static-fns
                                :external-libs external-libs
                                :verbose verbose
                                :context (keyword context)})]
     (! js/window.COMPILED true); for some reason it is required with read-eval-call
-    (replumb/read-eval-call opts cb s)))
+    (replumb/read-eval-call opts cb (str preamble s))))
 
 (defn my-eval [{:keys [file source file lang name path cache] :as args}]
   (cljs/js-eval args))
 
-(defmethod core-eval :core [s {:keys [static-fns context external-libs verbose] :or {static-fns false context nil external-libs nil}} cb]
+(defmethod core-eval :core [s {:keys [preamble static-fns context external-libs verbose] :or {preamble nil static-fns false context nil external-libs nil}} cb]
```

Comment:
It's here to allow me to dispatch to either :replumb or :core at dev time - to make sure that both works.
I'm thinking of getting rid of replumb some time